src/Employer/Employer.Web/ViewModels/Part2/Skills/SkillsViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace Esfa.Recruit.Employer.Web.ViewModels.Part2.Skills
+{
+    public class SkillsViewModel : SkillsEditModel
+    {
+        public List<SkillViewModel> Column1Checkboxes { get; set; }
+        public List<SkillViewModel> Column2Checkboxes { get; set; }
+        public List<string> CustomSkills { get; set; }
+    }
+
+    public class SkillViewModel
```

Comment:
Suggestion: Maybe call this `Skill` and move to the Web.Models folder. That way the Skills service doesn't reference the class with ViewModel in the name.

Reply:
I did think about making this service less view centric but as it is UI aware with regards to column1 and column2 I didn't mind this class being bound to the ViewModel.